rollup.config.js

```diff
@@ -46,7 +46,20 @@ export default [
       file: 'dist/esm/tiny-invariant.js',
       format: 'esm',
     },
-    plugins: [typescript({ module: 'ESNext' })],
+    plugins: [
+      typescript({ module: 'ESNext' }),
+      // https://github.com/rollup/rollup/blob/69ff4181e701a0fe0026d0ba147f31bc86beffa8/build-plugins/emit-module-package-file.ts
+      {
+        generateBundle() {
```

Comment:
would that call `generateESMPackageJson` or something instead of `generateBundle`?

Reply:
hm, this is just a plugin hook - it has to use a specific name to be called at the appropriate moment

Reply:
Ah ok, didn't know that!